pkg/aggregator/mutating_walker_test.go

```diff
@@ -52,22 +79,32 @@ func fuzzFuncs(f *fuzz.Fuzzer, refFunc func(ref *spec.Ref, c fuzz.Continue, visi
 			*url = spec.SchemaURL("http://url")
 		},
 		func(s *spec.Swagger, c fuzz.Continue) {
-			// fuzz everything first with invisible==true
-			invisible++
-			c.FuzzNoCustom(s)
-			invisible--
+			enter(s, false, c)
+			defer leave(false)
 
 			// only fuzz those fields we walk into with invisible==false
 			c.Fuzz(&s.Parameters)
 			c.Fuzz(&s.Responses)
 			c.Fuzz(&s.Definitions)
 			c.Fuzz(&s.Paths)
 		},
+		func(p *spec.PathItem, c fuzz.Continue) {
```

Comment:
this one was missing before, the walker does not mutate the ref if there is one in a `PathItem`.

Reply:
split into another commit.